Makefile

```diff
@@ -0,0 +1,26 @@
+include .docker.env
+
+run-devserver:
+  @echo "Updating the docker env variables..."
+  @echo "save_datastore=$(save_datastore)"
+  @echo "disable_host_checking=$(disable_host_checking)"
+  @echo "no_auto_restart=$(no_auto_restart)"
+  @echo "prod_env=$(prod_env)"
+  @echo "maintenance_mode=$(maintenance_mode)"
+  @echo "source_maps=$(source_maps)"
```

Comment:
Can you indent these lines so the output is easier to read?

Reply:
done